applications/GeoMechanicsApplication/custom_constitutive/coulomb_yield_surface.cpp

```diff
@@ -25,6 +25,12 @@ CoulombYieldSurface::CoulombYieldSurface(double FrictionAngleInRad, double Cohes
 {
 }
 
+double CoulombYieldSurface::GetFrictionAngleInRad() const { return mFrictionAngle; }
```

Comment:
```suggestion
double CoulombYieldSurface::GetFrictionAngleInRadians() const { return mFrictionAngle; }
```
Lengthier, but easier reading.

Reply:
thank you for changing.